power_grid_model_io_native_c/power_grid_model_io_native_c/include/power_grid_model_io_native_c/vnf_pgm_converter.h

```diff
@@ -28,7 +28,8 @@ PGM_IO_API PGM_IO_VnfConverter* PGM_IO_create_vnf_converter(PGM_IO_Handle* handl
  * @param converter_ptr A pointer to a PGM_IO_VnfConverter instace.
  * @return The pointer to the json string instance that holds data in PGM format.
  */
-PGM_IO_API char const* PGM_IO_get_pgm_input_data(PGM_IO_Handle* handle, PGM_IO_VnfConverter* converter_ptr);
+PGM_IO_API char const* PGM_IO_vnf_pgm_converter_get_input_data(PGM_IO_Handle* handle,
+                                                               PGM_IO_VnfConverter* converter_ptr);
```

Comment:
let's homogenize: either `make the type PGM_IO_VnfPgmConverter` or make the function name `PGM_IO_vnf_converter_*`